app/event/send-batch-processing-events.js

```diff
@@ -3,22 +3,26 @@ const raiseEventBatch = require('./raise-event-batch')
 
 const sendBatchProcessedEvents = async (paymentRequests, filename, sequence, batchExportDate) => {
```

Comment:
firstly, I will say -- I do not believe a malformed payment request can get to this line of code (as-is) in a way to cause it to break

I added the `.length` check and `?.map` check to ensure empty arrays and anything that isn't iterable would not send, i.e. 'this would pass .length but not ?.map'

Now tht I am writing this and looking at this, there are 2 potential error trails here: erroneous payment requests, i.e. 'strings', [], etc or an issue w/ `raiseEventBatch`

Based on this I think an approach to create a new function called `sendBatchProcessedEvent` which deals w/ the latter error checks; catching any issues thrown by `raiseEventBatch` and rejigging this to deal w/ the former error checks; catching all erroneous payment requests